tests/test_configuration.py

```diff
@@ -597,7 +596,6 @@ def test_get_set_rated_current(motion_controller):
 
 
 @pytest.mark.develop
-@pytest.mark.smoke
```

Comment:
Why this is not smoke?

Reply:
Theoretically because it takes more than 1 second. I'll check it again, just in case.